cysimdjson/jsoninter.h

```diff
@@ -155,9 +155,9 @@ inline dom::object to_object(dom::element & value, int * ok) {
 }
 
 
-inline std::string string_view_to_string(std::string_view sv) {
+inline const char * string_view_to_string(std::string_view sv) {
 	//TODO: This creates a string copy
-	return std::string(sv);
+	return std::string(sv).c_str();
```

Comment:
This is still incorrect since the lifecycle of the object ends.
You need to use https://docs.python.org/3/c-api/unicode.html functions ... check FastKafka for specific example. please.